pkg/resource/comparator.go

```diff
@@ -95,3 +102,102 @@ func (c *comparatorBuilder) BuildAsFunc() (reflect.Type, func(deployed resource.
 	c.Build()
 	return c.comparator.ResourceType, c.comparator.CompFunc
 }
+
+func containAllLabels(deployed resource.KubernetesResource, requested resource.KubernetesResource) bool {
+	deployedLabels := deployed.GetLabels()
+	requestedLabels := requested.GetLabels()
+
+	for key, value := range requestedLabels {
+		if deployedLabels[key] != value {
+			return false
+		}
+	}
+
+	return true
+}
+
+// CreateDeploymentConfigComparator creates a new comparator for DeploymentConfig using Trigger and RollingParams
+func CreateDeploymentConfigComparator() func(deployed resource.KubernetesResource, requested resource.KubernetesResource) bool {
+	return func(deployed resource.KubernetesResource, requested resource.KubernetesResource) bool {
+		dc1 := deployed.(*appsv1.DeploymentConfig)
```

Comment:
wouldn't be better raname these envc for a better readability?
i.e. dcRequest and dcDeployed?

Reply:
I'm not sure. We're following the `operator-utils` conventions and plus, the function name is already `...DeploymentConfig...`. I'm not a fan of big var names, though.

But if this is what the majority wants, I'd say to go ahead and rename it.

Reply:
@spolti @ricardozanini I have renamed the variables. Thanks.